e2e/atlas/events_test.go

```diff
@@ -66,8 +67,8 @@ func TestEvents(t *testing.T) {
 			atlasEntity,
 			eventsEntity,
 			"list",
-			"--orgId=5e429e7706822c6eac4d5970",
-			"--minDate=2020-04-01",
+			"--orgId="+os.Getenv("MCLI_ORG_ID"),
+			"--minDate="+time.Now().Add(-time.Hour*time.Duration(24)).Format("2006-01-02"),
```

Comment:
This test every now and then returns a 500 from the server, I'm guessing the time is too long now so I'm trying to make it just the last day